packages/language-css/src/embeds/css/__tests__/unit/media-queries.test.ts

```diff
@@ -0,0 +1,206 @@
+              ".bar{color:orange;}" +
+              "@media screen {.foo{color:blue;}}",
+          },
+          expected: [
+            {
+              content: "#id{font:monospace;}",
+              type: EMBED_TYPE_CSS,
+              startIndex: 30,
+              endIndex: 50,
+              getRaw(): string { return "#id{font:monospace;}"; },
+            },
+            {
+              content: ".foo{color:blue;}",
+              type: EMBED_TYPE_CSS,
+              startIndex: 85,
+              endIndex: 102,
+              getRaw(): string { return ".foo{color:blue;}"; },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      testName: "comments",
+      getScenario: () => [
+        {
+          file: {
+            type: "css",
+            content: "/*@media screen { .foobar{color:red;} }*/",
+          },
+          expected: [],
+        },
+        {
+          file: {
+            type: "css",
+            content: "/* \n @media screen { .foobar{color:blue;} }*/",
+          },
+          expected: [],
+        },
+        {
+          file: {
+            type: "css",
+            content: "/* \n @media screen { .foobar{color:red;} }*/" +
+              "@media screen { .foobar{color:blue;} }",
+          },
+          expected: [
+            {
+              content: " .foobar{color:blue;} ",
+              type: EMBED_TYPE_CSS,
+              startIndex: 59,
+              endIndex: 81,
+              getRaw(): string { return " .foobar{color:blue;} "; },
+            },
+          ],
+        },
+        {
+          file: {
+            type: "css",
+            content: "@media screen { .foobar{color:red;} }" +
+              "/* @media screen { .foobar{color:blue;} } */",
+          },
+          expected: [
+            {
+              content: " .foobar{color:red;} ",
+              type: EMBED_TYPE_CSS,
+              startIndex: 15,
+              endIndex: 36,
+              getRaw(): string { return " .foobar{color:red;} "; },
+            },
+          ],
+        },
+        {
+          file: {
+            type: "css",
+            content: "/* foo */@media screen { .foobar{color:red;} }/* bar */",
+          },
+          expected: [
+            {
+              content: " .foobar{color:red;} ",
+              type: EMBED_TYPE_CSS,
+              startIndex: 24,
+              endIndex: 45,
+              getRaw(): string { return " .foobar{color:red;} "; },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      testName: "edge cases, with matches",
+      getScenario: () => [
+        {
+          file: {
+            type: "css",
+            content: "@media screen { }",
+          },
+          expected: [
+            {
+              content: " ",
+              type: EMBED_TYPE_CSS,
+              startIndex: 15,
+              endIndex: 16,
+              getRaw(): string { return " "; },
+            },
+          ],
+        },
```

Comment:
Following #326, this test should ideally be changed (and the implementation accordingly) to verify that no embed is returned).

```suggestion
        {
          file: {
            type: "css",
            content: "@media screen { }",
          },
          expected: [],
        },
```

Reply:
Fixed in 9ea962ae96e0f7b62665bd0cb2c8563857e2446f